src/templates/finops-hub/modules/hub.bicep

```diff
@@ -62,6 +62,7 @@ var finOpsToolkitVersion = '0.0.1'
 
 resource defaultTelemetry 'Microsoft.Resources/deployments@2022-09-01' = if (enableDefaultTelemetry) {
   name: 'pid-${telemetryId}-${uniqueString(deployment().name, location)}'
+  tags: resourceTags
```

Comment:
No tags on this one. This is not really a release.

Reply:
removed